### ISSUE TITLE
[ONBOARDING] Libuv

### ISSUE THREAD

**Issue by jorydotcom** [openjs-foundation/cross-project-council#364]
## Project Onboarding Checklist - Libuv

**Stage: At Large**

- [ ] Adopt the OpenJS Foundation Code of Conduct
- [ ] Update project CoC reporting methods to include OpenJS Foundation escalation path
- [ ] Transfer official domains to OpenJS Foundation
- [ ] Identify and document other core project infrastructure
- [ ] If choosing to use a Contributor License Agreement (CLA) or Developer Certificate of Origin (DCO), make selection and implement appropriate tool
- [ ] Add or Update Governance.md document (required for Impact stage)
- [ ] Confirm required files in place (CODE_OF_CONDUCT.md, LICENSE.md)
- [ ] Project Charter is published on website or github
- [ ] Update legal copyright notice on project website and github
- [ ] Add OpenJS Foundation logo to project website
- [ ] Add Project logo to OpenJS Foundation website; update PROJECTS.md file
- [ ] Transfer logomark to the OpenJS Foundation
- [ ] If project is using crowdfunding platforms, add disclaimer to platforms
- [ ] Identify individuals from the project to join the CPC
- [ ] Document project and foundation contacts for:
  * marketing & social media
  * infrastructure
  * legal/governance help

**Comment by jorydotcom** [comment]
to-do: co-assign @addaleax & @bnoordhuis when they accept the team invite (or anyone else they would like to tag here)

**Comment by bnoordhuis** [comment]
Is this because we were a Node.js Foundation incubator project once?

**Comment by jorydotcom** [comment]
@bnoordhuis I'm not 100% sure of the Libuv's history with Node Foundation, but I think so. Libuv is included on the list of OpenJS Foundation projects so it's been included in my 'projects to onboard' list. If there's been an error here please let me know!

**Comment by bnoordhuis** [comment]
I don't want to speak for other libuv collaborators but I suspect we're not interested (I'm not at least) because the incubator status has had zero impact on the project and the connection between libuv and the NF isn't even tenuous, it's non-existent.

cc @cjihrig @saghul @santigimeno

**Comment by MylesBorins** [comment]
@bnoordhuis when the Node.js and JS foundations merged to become OpenJS the libuv project was part of the assets transferred into the new foundation. This was something that was discussed with the team prior to things moving forward.

All projects in the foundation are going through this on-boarding process as part of making sure everything is in place. I'm p sure this will be almost 0 effort on the part of libuv, primarily it will involve going through the list and making sure everything is in place. I'm happy to help with this work if no one from the libuv project is interested

**Comment by bnoordhuis** [comment]
Okay, well, let's see what the other maintainers say. Personally, I don't see the point and talk of "part of the assets transferred" doesn't make me more receptive. That's implying it's a done deal when it's not.

**Comment by santigimeno** [comment]
Being part of the NF incubator means we have to make this "_transference_"?

I have to agree with Ben on this. I've not seen any impact (positive or negative) on being part of it so I'd rather not perform any move of the project into any place.

**Comment by cjihrig** [comment]
Does this have any impact on libuv having access to the Node.js CI? If it does, then I'm indifferent. If it doesn't, then I'm with Ben and Santi.

**Comment by saghul** [comment]
+1 to what Ben and Santi already pointed out.

**Comment by addaleax** [comment]
I also agree largely with what was already said – without clear benefits for libuv (like CI access), this probably doesn’t make sense.

> to-do: co-assign @addaleax & @bnoordhuis when they accept the team invite (or anyone else they would like to tag here)

I appreciate the ping but I also feel like I should point out that everybody else from the project who has commented so far here is (from my point of view) more involved and more invested in libuv than me.

**Comment by jorydotcom** [comment]
@addaleax my apologies, I just selected the first two maintainers on the maintainer list as opposed to inviting the whole maintainer team, I figured that would be simpler. I'll untag you on the issue.

It sounds like there is a deeper underlying conversation to have with the maintainership so I'll pause any further activity on this onboarding thread. How would you all like to follow-up in the meantime @bnoordhuis @saghul @MylesBorins @addaleax?

**Comment by cjihrig** [comment]
I can tell that my opinion isn't wanted here, but I'd just like to point out that it's pretty rude that even after Ben looped in the other maintainers, you still choose to ignore myself and @santigimeno 😄

**Comment by jorydotcom** [comment]
@cjihrig @santigimeno I assure you I meant no disrespect; of course you are part of this conversation. I didn't scroll up all the way to catch all the github accounts that have been pinged here as I was writing that last comment, so I accidentally left your handles off the list. I apologize for that mistake, and for making you feel as though your opinion wasn't wanted. I haven't had the opportunity to liaise with the Libuv team before, but I look forward to getting to know you all in some capacity! 🤗

**Comment by cjihrig** [comment]
It's OK @jorydotcom, and I apologize for the snarky response.
